src/modules/transactions/transactionsTypes/ERC721NFTTransaction.js

```diff
@@ -28,15 +31,12 @@ class ERC721NFTTransaction extends Transaction {
       senderAddress,
       receiverAddress,
       timestamp: Date.now(),
-      accountsAddresses: [senderAddress.toLowerCase()],
+      tokenType: TOKEN_TYPE_ERC_721,
+      accountsAddresses: [senderAddress.toLowerCase(), receiverAddress.toLowerCase()],
       ...receipt,
     }
-    /**
-     * TODO: Add action to save the transaction in the DB
-     */
-    return new Promise(resolve => {
-      resolve(transactionBody)
-    })
+    const transactionService = new TransactionService()
```

Comment:
i see that in other places you use `this.$services.TransactionService` should we use that everywhere instad of `new TransactionService()` ?